2주차/create-account/style.css

```
@@ -0,0 +1,125 @@
+body {
+  background: #F0F6FF;
+}
+
+.container {
+  margin-top :238px ;
```

```suggestion
  margin-top :238px;
```

위와 같이 맞춰주시면 좋을 것 같아요~! 전반적으로 코드 맞춰야하는 부분에 있어서 맞춰주시면 좋을 것 같습니다.
아니면 자동으로 포맷팅해주는 extension 을 설치해서 써보시면 좋을 것 같아요!
vscode 를 사용하신다면 [CSS Formatter](https://marketplace.visualstudio.com/items?itemName=aeschli.vscode-css-formatter), webstorm 도 분명 CSS Formatter 또는 Beautifier 같은거 있을거예요!

---

webstorm으로 설치했습니다.

---

수정했습니다!